workitemtype.go

```diff
@@ -15,7 +15,7 @@ var wellKnown = map[string]*models.WorkItemType{
 		Name: "system.workitem",
 		Fields: map[string]models.FieldDefinition{
 			"system.owner": models.FieldDefinition{Type: models.SimpleType{Kind: models.KindUser}, Required: true},
-			"system.state": models.FieldDefinition{Type: models.SimpleType{Kind: models.KindString}, Required: true},
+			"system.url":   models.FieldDefinition{Type: models.SimpleType{Kind: models.KindURL}, Required: true},
```

Comment:
Yeah, I understood now, as test failed :p
Modified this just for my dev-testing.
I will add table tests as you suggested. Not sure what exactly they are but will read and do :)